mungegithub/submit-queue/www/index.html

```diff
@@ -34,7 +34,7 @@ <h2 class="md-toolbar-tools">E2E Tests Failing. Entire submit queue blocked. Las
           <md-tab label="PRs" md-on-select="cntl.selectTab('prs')">
             <md-content class="md-whiteframe-z2">
               <md-toolbar>
-                <h2 class="md-toolbar-tools">Pull Requests</h2>
+                <h2 class="md-toolbar-tools">Pull Requests ({{cntl.prsCount}})</h2>
```

Comment:
cntrl.prs.length didn't work because prs is an object and not an array.  I initially tried to get the count without modifying the js, and this is the only solution that worked for me.  Regarding search, you're right that it's strange not to show the total count of the "displayed items".